swagger.json

```diff
@@ -6418,6 +6556,13 @@
             "in": "query",
             "type": "integer",
             "format": "int64"
+        },
+        "vehicleIdOrExternalId": {
+            "name": "vehicle_id_or_external_id",
+            "description": "ID of the vehicle.  This must be either the numeric ID or the external ID of the vehicle.",
```

Comment:
Could we elaborate that the numerical id is the Samsara generated identifier for the vehicle, and external ids are customer specified key=value pairs that can be defined for a vehicle using the PATCH endpoint?